src/components/Facility/CentralNursingStation.tsx

```diff
@@ -216,7 +223,7 @@ export default function CentralNursingStation({ facilityId }: Props) {
           {t("no_vitals_present")}
         </div>
       ) : (
-        <div className="3xl:grid-cols-3 mt-1 grid grid-cols-1 gap-1 lg:grid-cols-2">
+        <div className="mt-1 grid grid-cols-1 gap-1 lg:grid-cols-2 min-[2080px]:grid-cols-3">
```

Comment:
avoid hardcoding abritrary pixel values. prefer rem values over pixel values as pixel values may behave differently for different zoom settings.